backend/mutations.py

```diff
@@ -301,7 +313,33 @@ def mutate(self, info, id):
         return DeleteUserMutation(success=success)
 
 class ArtworkMetricsInput(graphene.InputObjectType):
+    total_visits = graphene.Int()       # number of visits to artwork
+    total_ratings = graphene.Int()      # number of ratings submitted to artwork
```

Comment:
Do we want to give the API this power? Seems like these should be read-only and auto-incremented directly using mongoengine

Reply:
Yeah, part of this update was refactoring to give less control to the frontend, I'll be editing other mutations as we go along taking away API powers because I initially set it up to be able to change and edit most things to prove that I could without errors. This would be a good place to start